modules/postgresql/outputs.tf

```diff
@@ -0,0 +1,16 @@
+output "username" {
```

Comment:
I think admin_username and admin_password would be better here, since they should not be used in applications because they have a high privilege